### ISSUE TITLE
Ship loadout missing internals for coriolis.io

### ISSUE THREAD

**Issue by nemesyssoft** [Marginal/EDMarketConnector#30]
The coriolis ship loadout is missing some components.
Here is the coriolis.io ship loadout generated by ED Market Connector:
```json
{
  "$schema": "http://cdn.coriolis.io/schemas/ship-loadout/2.json#", 
  "name": "Anaconda", 
  "ship": "Anaconda", 
  "components": {
    "standard": {
      "bulkheads": "Lightweight Alloy", 
      "cargoHatch": {
        "enabled": true, 
        "priority": 2
      }, 
      "powerPlant": {
        "class": 8, 
        "rating": "C", 
        "enabled": true, 
        "priority": 2
      }, 
      "thrusters": {
        "class": 7, 
        "rating": "D", 
        "enabled": true, 
        "priority": 1
      }, 
      "frameShiftDrive": {
        "class": 6, 
        "rating": "A", 
        "enabled": true, 
        "priority": 1
      }, 
      "lifeSupport": {
        "class": 5, 
        "rating": "D", 
        "enabled": true, 
        "priority": 1
      }, 
      "powerDistributor": {
        "class": 8, 
        "rating": "D", 
        "enabled": true, 
        "priority": 1
      }, 
      "sensors": {
        "class": 8, 
        "rating": "D", 
        "enabled": true, 
        "priority": 1
      }, 
      "fuelTank": {
        "class": 4, 
        "rating": "C", 
        "enabled": true, 
        "priority": 2
      }
    }, 
    "hardpoints": [
      null, 
      {
        "class": 3, 
        "rating": "C", 
        "enabled": true, 
        "priority": 1, 
        "group": "Beam Laser", 
        "mount": "Gimballed"
      }, 
      {
        "class": 3, 
        "rating": "C", 
        "enabled": true, 
        "priority": 1, 
        "group": "Beam Laser", 
        "mount": "Gimballed"
      }, 
      {
        "class": 3, 
        "rating": "C", 
        "enabled": true, 
        "priority": 1, 
        "group": "Beam Laser", 
        "mount": "Gimballed"
      }, 
      {
        "class": 2, 
        "rating": "F", 
        "enabled": true, 
        "priority": 1, 
        "group": "Pulse Laser", 
        "mount": "Turret"
      }, 
      {
        "class": 2, 
        "rating": "F", 
        "enabled": true, 
        "priority": 1, 
        "group": "Pulse Laser", 
        "mount": "Turret"
      }, 
      {
        "class": 1, 
        "rating": "E", 
        "enabled": true, 
        "priority": 1, 
        "group": "Fragment Cannon", 
        "mount": "Turret"
      }, 
      {
        "class": 1, 
        "rating": "E", 
        "enabled": true, 
        "priority": 1, 
        "group": "Fragment Cannon", 
        "mount": "Turret"
      }
    ], 
    "utility": [
      {
        "class": 0, 
        "rating": "A", 
        "enabled": true, 
        "priority": 1, 
        "group": "Shield Booster"
      }, 
      {
        "class": 0, 
        "rating": "A", 
        "enabled": true, 
        "priority": 1, 
        "group": "Shield Booster"
      }, 
      null, 
      null, 
      null, 
      null, 
      null, 
      null
    ], 
    "internal": [
      {
        "class": 7, 
        "rating": "E", 
        "enabled": true, 
        "priority": 2, 
        "group": "Cargo Rack"
      }, 
      {
        "class": 6, 
        "rating": "E", 
        "enabled": true, 
        "priority": 2, 
        "group": "Cargo Rack"
      }, 
      {
        "class": 6, 
        "rating": "E", 
        "enabled": true, 
        "priority": 2, 
        "group": "Cargo Rack"
      }, 
      {
        "class": 6, 
        "rating": "E", 
        "enabled": true, 
        "priority": 2, 
        "group": "Cargo Rack"
      }, 
      {
        "class": 5, 
        "rating": "A", 
        "enabled": true, 
        "priority": 1, 
        "group": "Prismatic Shield Generator"
      }, 
      {
        "class": 4, 
        "rating": "A", 
        "enabled": true, 
        "priority": 2, 
        "group": "Fuel Scoop"
      }, 
      {
        "class": 3, 
        "rating": "A", 
        "enabled": true, 
        "priority": 2, 
        "group": "Frame Shift Drive Interdictor"
      }, 
      {
        "class": 4, 
        "rating": "A", 
        "enabled": true, 
        "priority": 1, 
        "group": "Shield Cell Bank"
      }, 
      {
        "class": 1, 
        "rating": "C", 
        "enabled": true, 
        "priority": 2, 
        "group": "Scanner", 
        "name": "Advanced Discovery Scanner"
      }
    ]
  }
}
```

Here is the same ship loadout in ED Shipyard:
```
[Anaconda]
L: 3C/G Beam Laser
L: 3C/G Beam Laser
L: 3C/G Beam Laser
M: 2F/T Pulse Laser
M: 2F/T Pulse Laser
S: 1E/T Fragment Cannon
S: 1E/T Fragment Cannon
U: 0A Shield Booster
U: 0A Shield Booster

BH: 1I Lightweight Alloy
RB: 8C Power Plant
TM: 7D Thrusters
FH: 6A Frame Shift Drive
EC: 5D Life Support
PC: 8D Power Distributor
SS: 8D Sensors
FS: 5C Fuel Tank (Capacity: 32)

7: 7E Cargo Rack (Capacity: 128)
6: 6E Cargo Rack (Capacity: 64)
6: 6E Cargo Rack (Capacity: 64)
6: 6E Cargo Rack (Capacity: 64)
5: 5A Prismatic Shield Generator
5: 4A Fuel Scoop
5: 5C Fuel Tank (Capacity: 32)
4: 4C Fuel Tank (Capacity: 16)
4: 3A Frame Shift Drive Interdictor
4: 4A Shield Cell Bank
2: 1C Advanced Discovery Scanner
---
Cargo : 320 T
Fuel  : 80 T
```

**Comment by Marginal** [comment]
Confirm - it's getting confused by the extra Fuel Tanks.

**Comment by nemesyssoft** [comment]
Thanks for the update. No pressure but any idea when there will be an update?

-Laurent.
-- 
Laurent Daudelin							laurent@nemesys-soft.com <mailto:laurent@nemesys-soft.com>
AIM/iChat/Skype:LaurentDaudelin 				http://www.nemesys-soft.com/ <http://www.nemesys-soft.com/>
Logiciels Nemesys Software

**Comment by Marginal** [comment]
Fixed in Release 1.83.
